src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.css

```diff
@@ -168,3 +152,42 @@ a {
         color: #13709e;
         text-decoration: underline;
     }
+
+.showRawException {
+    font-size: 1.2em;
+    cursor: pointer;
```

Comment:
Since its a button now (but it doesn't show up as a button in the UI and it appears as a link) the cursor is needed to make it appear as a link.